extensions-core/druid-kerberos/src/main/java/io/druid/security/kerberos/KerberosAuthenticator.java

```diff
@@ -113,6 +122,7 @@ public KerberosAuthenticator(
       @JsonProperty("excludedPaths") List<String> excludedPaths,
       @JsonProperty("cookieSignatureSecret") String cookieSignatureSecret,
       @JsonProperty("authorizerName") String authorizerName,
+      @JsonProperty("name") String name,
```

Comment:
new property - please add to docs.

Reply:
this is injected by the Jackson from the list of authenticators. user does not set this

Reply:
sounds good.